src/cli/commands/run.js

```diff
@@ -83,9 +83,16 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
         const cmdWithArgs = stage === action ? sh`${unquoted(cmd)} ${args}` : cmd;
         const customShell = config.getOption('script-shell');
         if (customShell) {
-          await execCommand(stage, config, cmdWithArgs, config.cwd, String(customShell));
+          await execCommand({
+            stage,
+            config,
+            cmd: cmdWithArgs,
+            cwd: config.cwd,
+            isInteractive: true,
+            customShell: String(customShell),
+          });
         } else {
-          await execCommand(stage, config, cmdWithArgs, config.cwd);
+          await execCommand({stage, config, cmd: cmdWithArgs, cwd: config.cwd, isInteractive: true});
```

Comment:
Thanks for cleaning this up with named parameters! Can we do
```
await execCommand({
            stage,
            config,
            cmd: cmdWithArgs,
            cwd: config.cwd,
            isInteractive: true,
            customShell: customShell && String(customShell)
        });
```
and remove the conditional now? :)